### ISSUE TITLE
How is the result in the paper evaluated?

### ISSUE THREAD

**Issue by PROoshio** [microsoft/unilm#32]
I wanna to know the evaluation method of the generated task in the paper. Are you testing according to the highest validation set or following the results of the last epoch? If it is the second test method, is the random seed of each training fixed? Thank you so much~

**Comment by donglixp** [comment]
Hi @PROoshio ,

We used the first method you mentioned. Please refer to the code snippet as in https://github.com/microsoft/unilm/issues/31 . Moreover, the random seed was kept fixed.
